templates/puppet4.conf.j2

```diff
@@ -0,0 +1,30 @@
+# {{ ansible_managed }}
+[main]
+    # The Puppet log directory.
```

Comment:
Bits of this conf file are loaners from Puppet3 era and in the end we could probably prune most of them out. But for maintaining a diff during the transition, it's better to hold on to them for now.

Reply:
I also wouldn't be too surprised if we at some point may want to add more settings to this file :)
I'm sure it will get some love when needed.